fuzz/src/full_stack.rs

```diff
@@ -1063,9 +1063,630 @@ pub extern "C" fn full_stack_run(data: *const u8, datalen: usize) {
 	do_test(unsafe { std::slice::from_raw_parts(data, datalen) }, &logger);
 }
 
+fn ext_from_hex(hex_with_spaces: &str, out: &mut Vec<u8>) {
+	for hex in hex_with_spaces.split(" ") {
+		out.append(&mut <Vec<u8>>::from_hex(hex).unwrap());
+	}
+}
+
+fn two_peer_forwarding_seed() -> Vec<u8> {
```

Comment:
Move in separate commit would have been nice, but can filter out the real changes.

Reply:
Seeing it in the pull request on github is easier, imo